cmake/dependencies.cmake

```diff
@@ -1,6 +1,6 @@
 include_directories(${GRPC_DEPENDENCIES_DIR}/include)
 include_directories(${PROTOBUF_DEPENDENCIES_DIR}/src)
-include_directories(${GTest_INCLUDE_DIRS}/include)
+include_directories(${GTEST_DEPENDENCIES_DIR}/include)
```

Comment:
${GTEST_DEPENDENCIES_DIR}/include is too general.
make it a good named variable (like old one GTest_INCLUDE_DIRS) and set it properly